src/acceptance/java/io/codurance/pouch/ListResourcesFeature.java

```diff
@@ -58,14 +56,18 @@ public void shouldListResources() throws JsonProcessingException, JSONException
         databaseHelper.insertResource(resourceTwo);
         databaseHelper.insertResource(resourceThree);
 
-        var actual = when().get("/resources")
-                .getBody()
-                .asString();
-
-        var objectMapper = new ObjectMapper();
-
-        var expected = objectMapper.writeValueAsString(asList(resourceOne, resourceTwo, resourceThree));
-
-        JSONAssert.assertEquals(expected, actual, false);
+        when().get("/resources")
+                .then()
+                .statusCode(SC_OK)
+                .contentType(JSON)
+                .body("[" + (resourceOne.getId()-1) + "].added", equalTo("2018-12-05 16:01:00.0"))
+                .body("[" + (resourceOne.getId()-1) + "].title", equalTo("Spring Data JDBC"))
+                .body("[" + (resourceOne.getId()-1) + "].url", equalTo("https://spring.io/projects/spring-data-jdbc"))
+                .body("[" + (resourceTwo.getId()-1) + "].added", equalTo("2018-12-05 16:02:00.0"))
+                .body("[" + (resourceTwo.getId()-1) + "].title", equalTo("SQL Fiddle"))
+                .body("[" + (resourceTwo.getId()-1) + "].url", equalTo("http://sqlfiddle.com/"))
+                .body("[" + (resourceThree.getId()-1) + "].added", equalTo("2018-12-05 16:03:00.0"))
+                .body("[" + (resourceThree.getId()-1) + "].title", equalTo("PostgreSQL: The world's most advanced open source database"))
+                .body("[" + (resourceThree.getId()-1) + "].url", equalTo("https://www.postgresql.org/"));
```

Comment:
Since @marc0der and I decided to go with REST Assured instead of JSONAssert, we'd reintroduce some magic numbers into the assertions. Those calls you see here are an attempt to make the indices of the data less arcane.